scripts/jenkins/ardana/ansible/init.yml

```diff
@@ -131,6 +132,25 @@
     become: true
     become_user: ardana
 
+  - name: Prepare for cobbler deployment
+    shell: |
+        ansible-playbook -vvv -i hosts/localhost cobbler-deploy.yml -e "ardanauser_password=linux"
+
+    when: usecobbler is defined and usecobbler|bool == True
+    args:
+      chdir: /var/lib/ardana/openstack/ardana/ansible
+    become: true
+    become_user: ardana
+
+  - name: Rebuild nodes for cobbler deployement
+    shell: |
+        ansible-playbook -vvv -i hosts/localhost bm-reimage.yml  ardanauser_password="linux"
```

Comment:
"ardanauser_password=linux" like in line 137?

Reply:
Don't you need `-e ardanauser....` to pass that value into the environment?